### DISCUSSION TITLE
My .msi works fine until I put it in a bootstrapper

### DISCUSSION THREAD

**Discussion by ivanSKYG** [wixtoolset/issues#6879] [Questions]
I make an .msi using wix, then bundle it into a bootstrapper with wix. When run by itself the .msi runs quickly, and doesn't ask the user for any input. When I bundle it into a bootstrapper, I get popups while it installs the .msi stating "Are you sure you want to cancel?" with yes/no buttons. Clicking no results in a successful installation. Why am I getting these popups? Are there pitfalls I should be aware of? Any help would be appreciated.

Even if I just put the .msi into the bootstrapper I run into this issue. I'm using wix 3.11.2

My simplified bootstrapper with redactions:
`
<?xml version="1.0"?>
    <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
      xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
      <Bundle
        Name="redacted"
        Version="$(var.Version)"
        Manufacturer="redacted, Inc."
        Copyright="© redacted"
        AboutUrl="https://redacted.app/"
        UpgradeCode="redacted-8e86-422f-b7ee-redacted"
        Condition="VersionNT >= v5.1">
      <BootstrapperApplicationRef
        Id="WixStandardBootstrapperApplication.HyperlinkLicense">
        <bal:WixStandardBootstrapperApplication 
          LicenseUrl=""
          LogoFile="redactedIcon64x64.png"
          ShowVersion="yes"
          SuppressRepair="yes"
          SuppressOptionsUI="yes" />
      </BootstrapperApplicationRef>
      <Chain>      
        <MsiPackage
          SourceFile="redacted.msi"/>
      </Chain>
      </Bundle>
    </Wix>
`

The CLI commands:
`candle .\myBootstrapper.xml -ext WixBalExtension -ext WixUtilExtension -dVersion="1.0.4470"`
`light .\myBootstrapper.wixobj -ext WixBalExtension -ext WixUtilExtension -sw1076 -o setup.exe`

**Comment by robmen** [comment]
You'll want to [look at the log files](https://support.firegiant.com/hc/en-us/articles/230912407-Create-a-log-file-) to begin diagnosing what is going awry.
